charts/acp/values.yaml

```diff
@@ -81,108 +92,265 @@ ingress:
 
   ## Ingress TLS configuration
   ## Secrets must be manually created in the namespace
+  ##
   tls: []
   #  - secretName: acp-tls
   #    hosts:
   #      - acp.domain.com
 
+## Deployment annotations
+##
+# annotations: {}
+
+## Autoscaling parameters
+##
 autoscaling:
+  ## Enable autoscaling
+  ##
   enabled: false
+  
+  ## Define mix replica count
+  ##
+  # minReplicas: 0
+
+  ## Define max replica count
+  ##
+  # maxReplicas: 1
+  
+  ## The average CPU usage of a all pods in a deployment
+  ##
+  # targetCPUUtilizationPercentage: ""
+  
+  ## The average memory usage of a all pods in a deployment
+  ##
+  # targetMemoryUtilizationPercentage: ""
 
 ## Number of ACP replicas to deploy
 ##
 replicaCount: 1
 
-## acp resource requests and limits
+## Pod annotations
+##
+# podAnnotations: {}
+
+## Pod security context
+##
+# podSecurityContext: {}
+
+## ACP resource requests and limits
 ## Ref: http://kubernetes.io/docs/user-guide/compute-resources/
 ##
 resources: {}
 
+## ACP node selector
+##
+# nodeSelector: {}
+
+## ACP pod affinity
+##
+# affinity: {}
+
+## ACP pod tolerations
+##
+# tolerations: {}
+
 ## A security context defines privilege and access control settings for a Pod or Container
 ## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+##
 securityContext:
   runAsUser: 0
 
-# ACP feature flags
+## ACP feature flags
```

Comment:
eventually, we should drop the features from values.yaml completely and redirect users to reference.yaml as we discussed

Reply:
https://cloudentity.atlassian.net/browse/DEVOPS-1585